src/org/exist/xquery/functions/fn/FunUnparsedText.java

```diff
@@ -20,6 +20,7 @@
 
 package org.exist.xquery.functions.fn;
 
+import com.googlecode.junittoolbox.ParallelRunner;
```

Comment:
in a non-@Test class?

Reply:
Oops, that went into the wrong class. I have fixed that now.